README.md

```diff
@@ -1,84 +1,75 @@
-# Group Income (Simple Edition)
+<p align="center"><img width="400px" src="frontend/assets/images/logo-transparent.png" alt="Group Income (Simple Edition)"></p>
 
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 [![Gitter](https://badges.gitter.im/okTurtles/group-income.svg)](https://gitter.im/okTurtles/group-income) [![Build Status](https://img.shields.io/travis/okTurtles/group-income-simple/master.svg)](https://travis-ci.org/okTurtles/group-income-simple) [![Deps](https://david-dm.org/okTurtles/group-income-simple.svg)](https://david-dm.org/okTurtles/group-income-simple/#info=dependencies) [![Dev Deps](https://david-dm.org/okTurtles/group-income-simple/dev-status.svg)](https://david-dm.org/okTurtles/group-income-simple/#info=devDependencies) [![Donate](https://img.shields.io/badge/donate%20-%3D%E2%9D%A4-blue.svg)](https://okturtles.org/donate/)
 
+## About
 [Group Income](https://groupincome.org/) is a fair income sharing mechanism that allows groups to provide their members a minimum income.
 
 This semi-centralized "Simple Edition" of Group Income is our way to rapidly prototype, develop, and research the concept with real groups.
 
-## Installation
+## Getting Started
 
 1. Install [Node.js](https://nodejs.org) (version 8 or greater).
 2. Install [Grunt](https://github.com/gruntjs/grunt): `npm install -g grunt-cli`
 3. Clone this repo (or a fork of it if you plan on [contributing](#contributing)) and `cd` into it.
 4. Install dependencies: `npm install`
+5. Now try out the [dev workflow](#basic-workflow).
+- 👩‍🎨 Check out how Group Income will feel like by taking a look at the [design files on Figma](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=1204%3A0)
 
-Now try out the [dev workflow](#basic-workflow).
-
----
+## Contributing
 
 We use [standard](https://github.com/feross/standard) for the code style and [Github project boards](https://help.github.com/articles/about-project-boards/) for efficient project management.
 
 [![js-standard-style](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)
 
-## Contributing
+We are committed to ensuring digital accessibility for all people. We are continually improving the user experience for everyone, and applying the relevant accessibility standards. We will be conformant with [WCAG 2.0](https://www.w3.org/WAI/standards-guidelines/wcag/) Level AA and trying or best to reach Level AAA in some areas of the application.
+
+<a href="https://www.w3.org/WAI/WCAG2AA-Conformance"
+  title="Explanation of WCAG 2.0 Level Double-A Conformance">
+  <img height="32" width="88"
+    src="https://www.w3.org/WAI/wcag2AA"
+    alt="Level Double-A conformance, W3C WAI Web Content Accessibility Guidelines 2.0">
+</a>
```

Comment:
Excellent idea to add this section. I would remove the first sentence though (the one starting with `We are committed...`), for two reasons:

1. It is somewhat redundant because of the second sentence.
2. Ensuring accessibility for "all people" is a very high bar, and one we can strive for, but to commit ourselves to it is to make a promise that we might not be able to keep, and I don't want anyone upset over broken promises. :)

Reply:
I understand. Removed